src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.label {
+  position: relative;
+  display: block;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+}
+
+.small_hight {
+  height: 42px;
+}
+
+.search {
+  position: absolute;
+  font-family: inherit;
+  border: none;
+  outline: none;
+  padding: 0;
+  padding-left: 62px;
+  font-size: 16px;
+  font-weight: 900;
+  right: 0%;
+  top: 0%;
+  left: 0%;
+  bottom: 0%;
+}
+
+.small_search {
+  padding-left: 33px;
+  font-size: 14px;
+}
```

Comment:
[Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2-nd point